src/spec/integration/channels.spec.coffee

```diff
@@ -29,11 +29,15 @@ describe 'Integration Channels', ->
   channels = []
 
   beforeEach (done) ->
-    @client = new SphereClient config: Config
+    @client = new SphereClient
+      config: Config
+      stats:
+        includeHeaders: true
 
     @client.channels.save(newChannel())
     .then (result) =>
       expect(result.statusCode).toBe 201
+      expect(result.http.response.headers['x-correlation-id']).toBeDefined()
```

Comment:
we should use the same case for the header as we return from nginx (`X-Correlation-ID`)